packages/core/src/parse/figma.ts

```diff
@@ -206,7 +209,7 @@ export function convertFigmaVariablesFormat(manifest: FigmaVariableManifest, opt
   }
 
   // 3. explode flat structure into nested structure
-  for (const [id, token] of Object.entries(dtcgFlat)) {
+  for (const id in dtcgFlat) {
```

Comment:
Here, and in other places, the performance crept up, so I removed a lot of array creation bloat with `Object.keys()` and `Object.entries()`.

It may not fix the slow test problem, but it should be a _lot_ more memory-friendly